apps/remote_control/lib/lexical/remote_control/build/progress.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Lexical.RemoteControl.Build.Progress do
+  alias Lexical.RemoteControl
+
+  import Lexical.RemoteControl.Api.Messages
+
+  def with_progress(label, func, opts \\ []) when is_function(func) do
```

Comment:
note: you can specify the arity of the function in `is_function`.
```suggestion
  def with_progress(label, func, opts \\ []) when is_function(func, 0) do
```